UnitySetup/UnitySetup.psm1

```diff
@@ -1870,7 +1870,20 @@ function Start-UnityEditor {
             $process.Start() | Out-Null
 
             if ( $Wait ) {
-                $process.WaitForExit()
+                if( $LogFile ) {
+                    New-Item -Path $LogFile -ItemType File -Force
+                    $ljob = Start-Job -ScriptBlock { param($log) Get-Content "$log" -Wait } -ArgumentList $LogFile
+
+                    while ( -not $process.HasExited )
+                    {
+                        Receive-Job $ljob
+                    }
+
+                    Receive-Job $ljob -Wait -AutoRemove
```

Comment:
yeah this is never completing for some reason and my CI step is hanging here.

Reply:
Ah, my bad! `Get-Content -Wait` never exits so this job would never exit. =(

Reply:
I think you could loop on Receive-Job until it fails to return any results though that's a bit of a race too. =(

Reply:
I don't mind that we don't get all the output. We're saving the log to disk anyway and publishing it in the artifacts. Really having it in the console is for convenience.

Reply:
Yeah, our CI writes the whole thing out too, but just after Unity has exited. Super convenient. I love this approach where we could watch it in action though. =)